src/pyuvdata/utils/times.py

```diff
@@ -369,10 +372,21 @@ def _select_times_helper(
         Length 2 tuple giving (rtol, atol) to use for time matching.
     lst_tols : tuple of float
         Length 2 tuple giving (rtol, atol) to use for lst matching.
+    time_inds : array_like of int, optional
+        The btime indices to keep in the object. This is not commonly used.
```

Comment:
btime -> time

Reply:
Fixed in bee13af03969accbf712e9db80f799bd0d142a9b